packages/react-components/react-shared-contexts/library/src/CustomStyleHooksContext/CustomStyleHooksContext.ts

```diff
@@ -23,6 +23,7 @@ export type CustomStyleHooksContextValue = Partial<{
   usePresenceBadgeStyles_unstable: CustomStyleHook;
   useButtonStyles_unstable: CustomStyleHook;
   useCompoundButtonStyles_unstable: CustomStyleHook;
+  useKeytipsStyles_unstable: CustomStyleHook;
```

Comment:
please undo these changes and manually add this to within Keytips.tsx etc

https://github.com/microsoft/fluentui/pull/31995/files#diff-d7f3107892be3bbccf7579fb77ca3a5a090c5fee8c1e145afb508e961e13859eR16-R21